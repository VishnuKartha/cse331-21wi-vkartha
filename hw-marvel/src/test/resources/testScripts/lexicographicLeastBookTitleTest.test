# Tests that the algorithm favors the path with the lexicographical least book title if the child names are equal

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/alphabet2.tsv

LoadGraph alpha2 alphabet2.tsv

FindPath alpha2 A B

FindPath alpha2 B A