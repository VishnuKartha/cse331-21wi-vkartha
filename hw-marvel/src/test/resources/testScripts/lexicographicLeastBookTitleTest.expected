# Tests that the algorithm favors the path with the lexicographical least book title if the child names are equal

# To run this test, the marvel dataset must exist at
# hw-marvel/src/main/resources/data/alphabet2.tsv

loaded graph alpha2

path from A to B:
A to C via connection1
C to B via connection2

path from B to A:
B to C via connection2
C to A via connection1